crypto/keyring/keyring_test.go

```diff
@@ -37,16 +37,17 @@ func TestNewKeyring(t *testing.T) {
 	encCfg := simapp.MakeTestEncodingConfig()
 
 	// TODO fix keyring.New in other packages
-	kr, err := keyring.New("cosmos", BackendFile, dir, encCfg.Marshaler, mockIn)
+	// TODO fix line 41 cannot use encCfg.Marshaler (variable of type codec.Codec) as io.Reader value in argument to keyring.New: missing method Readcompiler 
```

Comment:
@amaurym any idea how to fix an error at line 40?

Reply:
You can use nil for the io.Reader argument in tests